### DISCUSSION TITLE
Node Group Input parameters lost their min max clamp

### DISCUSSION THREAD

**Discussion by HSDSZ** [bnpr/Malt#403] [Feedback & Feature Requests]
### Malt version

node-groups-latest

### Blender version

Blender 3.30

### OS

Windows 11 64bits

### Hardware info

Intel i5-9400F | 8 GB RAM | NVDIA GTX 1660 Ti

### Issue description and reproduction steps

![image](https://user-images.githubusercontent.com/36077492/192550521-141ce54b-f054-4030-9fea-fa8a0161ad11.png)

For example, the Size input, should be within 0 to 1. But after you group it, it could be changed to any value

### Attachments

[untitled.zip](https://github.com/bnpr/Malt/files/9656756/untitled.zip)

**Comment by pragma37** [comment]
I move this to Feedback and Feature Requests since this is not really a bug.

While I see the point, what would happen if you plug the Size socket into multiple nodes, for example?

This would also require updating all the node trees that use a certain group whenever their Input node is updated, which can be slow.

**Reply by HSDSZ** [reply]
Ah, I see. You are right. 
Then, maybe trying the blender shader solution, by fixing the min max by user
![image](https://user-images.githubusercontent.com/36077492/192585590-8ea779e6-664c-414f-9c3e-5302c7be27f8.png)

Well, this is not an urgent issue.

**Reply by pragma37** [reply]
That seems like a nicer solution.
